Support allowed scheme with `openUrl` card action (#3226)
microsoft/BotFramework-WebChat: CHANGELOG.md

## [Unreleased]

### Breaking changes

> -  Affecting Adaptive Card, legacy card and suggested actions

Reviewer: ```suggestion
-  Affecting Adaptive Cards, legacy cards and suggested actions
```

>    -  For `openUrl` card action, we are now whitelisting the URL scheme using the same set of whitelist from the default Markdown + sanitize engine, it includes `data`, `http`, `https`, `ftp`, `mailto`, `sip`, and `tel`

Reviewer: ```suggestion
   -  For `openUrl` card action, we are now whitelisting the URL scheme using the same whitelist from the default Markdown + sanitize engine, which includes `data`, `http`, `https`, `ftp`, `mailto`, `sip`, and `tel`
```
